frontend/packages/app/src/lib/components/room-settings/room-settings.svelte

```diff
@@ -99,6 +109,18 @@
         on:activate={onAutoRevealChange}
     />
 
+    <label>
+        <span class="time-box-label">Timebox in s</span>
+        <input 
+        class="select" 
+        type="number" 
+        min="0"  
+        step="5" 
+        disabled={status === Status.InProgress}
+        on:change={onTimeboxChange} 
+            value={timeboxInSeconds} />
```

Comment:
nit: the closing `/>` should be in a new line to see we the end of the element is